bouncer/tests/broker_fee_collection_test.ts

```diff
@@ -29,11 +31,24 @@ const swapAssetAmount = {
   [Assets.USDC]: 1000,
 };
 const commissionBps = 1000; // 10%
+const cfNodeEndpoint = process.env.CF_NODE_ENDPOINT ?? 'ws://127.0.0.1:9944';
 
-const chainflip = await getChainflipApi();
+const chainflip = await ApiPromise.create({
```

Comment:
Why can we not just use `getChainflipApi` here anymore?

Reply:
We need those types definition added to the API. 
I just added it to getChainflipApi so we can directly use that.